### DISCUSSION TITLE
Welcome to ComposeCookBook Discussions!

### DISCUSSION THREAD

**Discussion by Gurupreet** [Gurupreet/ComposeCookBook#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hellosagar** [comment]
First of all thanks for your great effort in starting this repo.

I have a questions
I tried the Beta 05 release apk on my device Redmi 8, but it's laggy even the ripple on button is lagging. 
Any idea why it that and even compose claim must better perform than XML but apk is saying something else
